### ISSUE TITLE
[BUG] UMAPINFO label = clear syntax no longer works in 4.13.0

### ISSUE THREAD

**Issue by ravenlikeadesk** [ZDoom/gzdoom#2774]
### GZDoom version

4.13.0

### Which game are you running with GZDoom?

Doom 2

### What Operating System are you using?

Windows 10

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

The following syntax in UMAPINFO now results in an error on map load in gzdoom 4.13.0
```
MAP map01
{
	label = clear
	music = "D_RUNNIN"
}
```
The error is the following:
```
G_ParseMapInfo: Load map definitions.

Execution could not continue.
Script error, "sleepwalking.wad:UMAPINFO" line 3:
Expected string constant but got identifier 'clear' instead.
```

It used to work in previous versions and it also works in other source ports (e.g. dsda-doom).

The syntax is also valid according to UMAPINFO spec: https://github.com/kraflab/umapinfo/blob/master/docs/spec.md#label

### Steps to reproduce the behaviour.

Create empty pwad with the following UMAPINFO lump:
```
MAP map01
{
	label = clear
	music = "D_RUNNIN"
}
```
Or load existing pwad with such definition in umapinfo, e.g. https://drive.google.com/file/d/1qF4R-MI-5RZrKk8BZImRbV3PG3vHI7SS/view?usp=sharing (the wad that got this problem reported on).

### Your configuration

```ini
Fresh gzdoom 4.13.0 download, everything is default.
```

### Provide a Log

```
G_ParseMapInfo: Load map definitions.

Execution could not continue.
Script error, "sleepwalking.wad:UMAPINFO" line 3:
Expected string constant but got identifier 'clear' instead.
```

**Comment by RicardoLuis0** [comment]
those fields just weren't implemented at all in gzdoom prior to 4.13, seems the implementation was only partial
https://github.com/ZDoom/gzdoom/commit/2decf10863d6a44b69b747a431babb20885df0c1 @prof-hastig

**Comment by ravenlikeadesk** [comment]
I see, that explains it. Looks like now the implementation is not entirely correct though since it doesn't cover this case that the specification allows.

**Comment by RicardoLuis0** [comment]
indeed

**Comment by RicardoLuis0** [comment]
fixed with #2776
